src/libraries/System.Data.Odbc/src/Common/System/Data/Common/AdapterUtil.Odbc.cs

```diff
@@ -656,7 +656,9 @@ internal static int StringLength(string inputString)
             return ((null != inputString) ? inputString.Length : 0);
         }
 
+#if FEATURE_CER
```

Comment:
This file compiles for netcoreapp only. You can tell from this condition in the .csproj:
```
  <ItemGroup Condition="($(TargetFramework.StartsWith('$(NetCoreAppCurrent)')) or $(TargetFramework.StartsWith('netcoreapp2.0'))) and '$(TargetsAnyOS)' != 'true'">
```
-> this can be deleted.

Reply:
(multiple places)